### PR TITLE
AI UI refresh

### PR DESCRIPTION
**Pull request by ryanclark**
This refreshes the UI like in previous examples, and makes it compatible with the current theme.

image

image

To backport to v12, I'll just change the colors to be hardcoded to the dark theme.

### CODE REVIEW

File: web/packages/teleport/src/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -121,6 +123,8 @@ export function Navigation() {
       const category = getCategoryForRoute(features, next);
       const previousCategory = getCategoryForRoute(features, location);
 
+      console.log('category', features, previousCategory);
```
Comment (jakule) [inline web/packages/teleport/src/Navigation/Navigation.tsx:126]: The same here, we're starting to port this back to master, so we should stop leaving debug code here.

File: web/packages/teleport/src/Navigation/Navigation.tsx [hunk 2]
```diff
@@ -158,6 +162,8 @@ export function Navigation() {
         return;
       }
 
+      console.log('');
```
Comment (jakule) [inline web/packages/teleport/src/Navigation/Navigation.tsx:165]: The same as above

File: web/packages/teleport/src/Navigation/NavigationSwitcher.tsx [hunk 3]
```diff
@@ -134,6 +143,10 @@ export function NavigationSwitcher(props: NavigationSwitcherProps) {
     assistEnabled
   );
 
+  const theme = useTheme();
+
+  console.log(theme);
```
Comment (jakule) [inline web/packages/teleport/src/Navigation/NavigationSwitcher.tsx:148]: The same

**Review by jakule — APPROVED** [review]

File: web/packages/teleport/src/Navigation/Navigation.tsx (hunk 1 above)
Comment (ryanclark) [reply web/packages/teleport/src/Navigation/Navigation.tsx:126]: My bad, I forgot to do a sweep :D

**Pull request merged**